LightBlue/Infrastructure/HostRunner.cs

```diff
@@ -32,6 +32,12 @@ public class HostRunner
 
         private void RunRole(Type workerRoleType)
         {
+            if (workerRoleType.Name == "Program")
```

Comment:
Although this is a pretty strong convention, we should probably use Assembly.GetEntryPoint()